### ISSUE TITLE
Latitude and longitude are inverted in the detail bar on the map

### ISSUE THREAD

**Issue by rcavaliere** [noi-techpark/it.bz.opendatahub.analytics#70]
See screenshot. The values associated to latitude and longitude are inverted
![Screenshot from 2021-02-27 06-07-35](https://user-images.githubusercontent.com/9825179/109376063-31d58f00-78c2-11eb-9565-e846aea712a6.png)

**Comment by noctho** [comment]
You are right. These two values are inverted, but this bug only affects the detail bar. I checked this and fixed it with 0f8eae7588f8323b19c64c8aa33859c505771eb2.

I can see that my last recent PR (#69) is still open, so it contains already this commit in it.
